src/ephemeral.rs

```diff
@@ -0,0 +1,253 @@
+//! # Ephemeral messages
```

Comment:
```suggestion
//! # Ephemeral and auto-delete messages
//!
//! Device auto-deletion is a global setting which will delete messages locally
//! after a configured delay.
//!
//! Ephemeral messages can be enabled per-chat and coordinates an expiry
//! with all the participants of the chat.  When enabled a system-message is
//! sent out announcing the expiry timeout and each subsequent message to
//! the chat will also include this expiry timeout.
//!
//! In both cases the implementation deletes messages on access of the chatlist
//! or the chat.  Messages are not instantly deleted, instead moved to the trash
//! and overwritten so that the IMAP jobs can still see the messages and remove
//! them from the server as well at a later point.
```
Is this more or less correct?